src/app/admin/admin-reports/filtered-collections/filtered-collection.model.ts

```diff
@@ -0,0 +1,36 @@
+export class FilteredCollection {
```

Comment:
Similar to Java, we require TypeDocs (similar to JavaDocs) to be added to all exported/public classes.  If you could add them to all the new classes in this PR, that'd be appreciated (they don't need to be added to HTML or SCSS files...just the TS module files).